style.css

```diff
@@ -63,4 +63,372 @@
   .icons {
     padding-right: 1.5rem;
   }
+
+  .projects {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 1rem;
+    gap: 0.75;
+    width: 20.438rem;
+    height: 34.5rem;
+    background: #fff;
+    border: 0.063rem solid #dfe1e6;
+    border-radius: 1rem;
+  }
```

Comment:
- [ ] As mentioned previously please make sure to stick with the given design for the fonts as you should use the "Poppins" font for the texts in the project card. You can easily download it from google fonts.
- Your design:
  ![image](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/104968236/accd81a5-7d32-47e9-a615-ec2767eb6e23)
- Figma design:
  ![image](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/104968236/c8f9408a-f111-4376-bb96-5acf92472fec)
- [ ] This means you have to remove the `font-family` you've added to each text in the project card that's not "Poppins". ✔